.gitignore

```diff
@@ -1,7 +1,7 @@
 *.DS_Store
 *.a
 *.picklecache
-*.pyc
+*.py[cd]
```

Comment:
This has nothing to do with the description of the PR.m Please revert or include this in a commits of its own with a suitable commit message.

Reply:
python on windows use `pyd` as binary extension ext name, why it's not related in this PR?

Reply:
Because the title of the PR is "fix parser test on windows" and this does not fix any test.

Reply:
this is easy to fix

Reply:
How do you get a `.pyd` file in the source directory?

Reply:
because I call meson manually

Reply:
I'm getting tired of your half answers. If you invoke meson manually you do not have build artifacts in the source directory. The only way to get a `.pyd` file in the source directory is if you copy it there.

What about you don't do that and instead use an editable install?
```
$ python -m pip install --no-build-isolation -e .
```
This is the recommended way to install a project for development.

Reply:
This hunk needs to be its own commit with a suitable commit message.